controller/src/beerocks/master/tasks/bml_task.h

```diff
@@ -83,6 +89,7 @@ class bml_task : public task {
         ACS_START_EVENT_AVAILABLE,
         CSA_NOTIFICATION_EVENT_AVAILABLE,
         CAC_STATUS_CHANGED_NOTIFICATION_EVENT_AVAILABLE,
+        OPERATIONAL_STATUS_UPDATE,
```

Comment:
IMO we should call TOPOLOGY_RESPONSE, the BML triggers topology query and waits for topology response, the name operational is just for the command.

Reply:
done